api/src/services/users.ts

```diff
@@ -147,7 +155,7 @@ export class UsersService extends ItemsService {
 		const user = await this.knex
 			.select('id', 'status')
 			.from('directus_users')
-			.where({ email })
+			.where('email', 'like', `%${email}%`)
```

Comment:
Seeing we explicitly convert everything to lowercase elsewhere, this shouldn't be needed.